utils.go

```diff
@@ -83,7 +83,8 @@ func PostUrlUtil(longUrl string, alias string, validity int) (result urlStruct)
 				switch pair.LongURL {
 				case "":
 					// If custom url is available create a new entry with it
-					result.ShortURL = SaveUrl(longUrl, alias, validity)
+					go SaveUrl(longUrl, alias, validity)
+					result.ShortURL = HOST_URL + alias
 				case longUrl:
 					// If custom url is already allocated for same long url then return the same
```

Comment:
This case should be removed. It has already been removed for single URL shortening. This is because it causes a discrepancy in the expiry date. One user sets the expiry date using this custom URL as 30 days and then another user comes and uses the same URL and sets the expiry to 2 days which is not right as one of the users will not get his intended expiry period. In fact, this is also the case for random short URLs. If we want the correctness of expiry dates then every user should also get a different random short URL as well.

Reply:
done